plugins.json

```diff
@@ -799,5 +799,14 @@
 		"min_version": "4.0.0",
 		"await_loading": true,
 		"variant": "both"
+	},
+	"sphere": {
+		"title": "PixelArt Sphere Generator",
+		"author": "hamburger2506",
+		"icon": "circle",
+		"description": "A plugin to create pixelated spheres, since minecraft does not allow real spheres due to something to do with rotation. Also I needed a sphere that I could do pixel art on to make it look like a planet. (Feel free to update or republish this plugin if you\'d like since I probably won\'t do mayor updates to it anymore.)",
+		"version": "1.0.0",
+		"variant": "both",
+		"tags": ['Minecraft: Java Edition']
```

Comment:
In the JSON meta data, `tags` should be `"tags": ["Minecraft: Java Edition"]`

Reply:
I fixed that in the newest pull request?